workflow/targets/phenotype.smk

```diff
@@ -203,8 +198,54 @@ PHENOTYPE_WILDCARDS = {
 
 PHENOTYPE_OUTPUTS_MAPPED = map_outputs(PHENOTYPE_OUTPUTS, PHENOTYPE_OUTPUT_MAPPINGS)
 
-PHENOTYPE_TARGETS = outputs_to_targets(
-    PHENOTYPE_OUTPUTS, PHENOTYPE_WILDCARDS, PHENOTYPE_OUTPUT_MAPPINGS
+PHENOTYPE_TARGETS_ALL = (
+    # Targets that need tiles
+    outputs_to_targets_with_combinations(
+        output_templates=PHENOTYPE_OUTPUTS["apply_ic_field_phenotype"],
+        valid_combinations=PHENOTYPE_VALID_COMBINATIONS,
+        extra_keys=PHENOTYPE_TILES
+    ) +
+    outputs_to_targets_with_combinations(
+        output_templates=PHENOTYPE_OUTPUTS["align_phenotype"],
+        valid_combinations=PHENOTYPE_VALID_COMBINATIONS,
+        extra_keys=PHENOTYPE_TILES
+    ) +
+    outputs_to_targets_with_combinations(
+        output_templates=PHENOTYPE_OUTPUTS["segment_phenotype"],
+        valid_combinations=PHENOTYPE_VALID_COMBINATIONS,
+        extra_keys=PHENOTYPE_TILES
+    ) +
+    outputs_to_targets_with_combinations(
+        output_templates=PHENOTYPE_OUTPUTS["identify_cytoplasm"],
+        valid_combinations=PHENOTYPE_VALID_COMBINATIONS,
+        extra_keys=PHENOTYPE_TILES
+    ) +
+    outputs_to_targets_with_combinations(
+        output_templates=PHENOTYPE_OUTPUTS["extract_phenotype_info"],
+        valid_combinations=PHENOTYPE_VALID_COMBINATIONS,
+        extra_keys=PHENOTYPE_TILES
+    ) +
+    # Targets that don't need tiles
+    outputs_to_targets_with_combinations(
+        output_templates=PHENOTYPE_OUTPUTS["merge_phenotype_info"],
+        valid_combinations=PHENOTYPE_VALID_COMBINATIONS
+    ) +
+    outputs_to_targets_with_combinations(
+        output_templates=PHENOTYPE_OUTPUTS["extract_phenotype_cp"],
+        valid_combinations=PHENOTYPE_VALID_COMBINATIONS,
+        extra_keys=PHENOTYPE_TILES
+    ) +
+    outputs_to_targets_with_combinations(
+        output_templates=PHENOTYPE_OUTPUTS["merge_phenotype_cp"],
+        valid_combinations=PHENOTYPE_VALID_COMBINATIONS
+    ) +
+    # Evaluation targets (plate level only)
+    outputs_to_targets_with_combinations(
+        output_templates=PHENOTYPE_OUTPUTS["eval_segmentation_phenotype"],
+        valid_combinations=[{"plate": combo["plate"]} for combo in PHENOTYPE_VALID_COMBINATIONS]
+    ) +
+    outputs_to_targets_with_combinations(
+        output_templates=PHENOTYPE_OUTPUTS["eval_features"],
+        valid_combinations=[{"plate": combo["plate"]} for combo in PHENOTYPE_VALID_COMBINATIONS]
+    )
```

Comment:
I wonder if there is a way to save on code here since we have a lot of repeat code? Not worth worrying about now imo